f8a_report/helpers/report_helper.py

```diff
@@ -72,8 +72,13 @@ def cleanup_tables(self, table_name, column_name, num_days):
             sql.Identifier(column_name)
         )
         logger.debug('Starting to clean up "%s" table', table_name)
-        # Executing query
-        self.cursor.execute(query, (num_days,))
+        try:
+            # Executing query
+            self.cursor.execute(query, (num_days,))
+        except Exception as e:
+            logger.error("cleanup failed with exception %r", e)
+
+        # Commiting
```

Comment:
```suggestion
try:
           # Executing query
            self.cursor.execute(query, (num_days,))
        except Exception as e:
            logger.error("cleanup failed with exception %r", e)
            return
           
```

Reply:
@prashbnair Are we leaving out the commit() statement?

Reply:
No, the commit is outside the exception handling code.

Reply:
yes,I have kept it outside.